### PR TITLE
Update dependencies

### LINKED ISSUE #1930: Update to Robolectric 3.2

### PR DESCRIPTION
**Pull request by WonderCsabo**
Also fixes #1930 .

### CODE REVIEW

File: AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/ServiceInjectionTest.java [hunk 1]
```diff
@@ -104,13 +104,13 @@ public void servicesAreInjected() {
 		assertThat(activity.uiModeManager).isNotNull();
 		// assertThat(activity.updateLock).isNotNull(); // hidden API
 		// assertThat(activity.usageStatsManager).isNotNull();  // TODO API 21
-		// assertThat(activity.usbManager).isNotNull(); // TODO no support yet in Robolectric
+		assertThat(activity.usbManager).isNotNull(); // TODO no support yet in Robolectric
```
Comment (dodgex) [inline AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/ServiceInjectionTest.java]: I think you should remove the TODO comments here and below?

File: AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/InjectExtraTest.java [hunk 2]
```diff
@@ -22,81 +22,113 @@
...
-	@Before
-	public void setUp() {
-		controller = Robolectric.buildActivity(ExtraInjectedActivity_.class);
-		activity = controller.get();
-	}
 
 	@Test
 	public void simpleStringExtraInjected() {
-		controller.withIntent(ExtraInjectedActivity_.intent(context).stringExtra("Hello!").get()).create();
+		ExtraInjectedActivity activity = Robolectric.buildActivity(ExtraInjectedActivity_.class, ExtraInjectedActivity_.intent(context)
```
Comment (dodgex) [inline AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/InjectExtraTest.java]: MAYBE extract the intent into an own variable to make the code a bit better readable?

**Comment by WonderCsabo** [comment]
@dodgex updated.

**Comment by dodgex** [comment]
Thank you! :)

**Pull request merged**